tools/archetype/binding.test/src/main/resources/archetype-resources/src/test/java/internal/__bindingIdCamelCase__InternalTest.java

```diff
@@ -1,5 +1,8 @@
+#set( $dt = $package.getClass().forName("java.util.Date").newInstance() )
```

Comment:
I'm not wondering whether we really still need this test class at all. The HandlerTest above actually is the much better example for a plain JUnit test.